geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -655,13 +677,22 @@ where
     Ok(inf_lines)
 }
 
+/// Voronoi Diagram Errors
 #[derive(Debug, PartialEq, Eq)]
 pub enum VoronoiDiagramError {
+    /// An error occurred when attempting to complete Delaunay Triangulation
+    /// before computing the Voronoi Diagram
     DelaunayError(DelaunayTriangulationError),
+    /// This error occurs when a conversion from a value to a Geo Generic T fails.
```

Comment:
" Geo generic `T` " ?

Reply:
Let me know if the update makes sense here pls.